### DISCUSSION TITLE
Ребрендинг – geekr.

### DISCUSSION THREAD

**Discussion by jarvis394** [jarvis394/geekr#145] [Show and tell]
### Ребрендинг

Теперь habra. - это geekr.

<img src="https://github.com/jarvis394/geekr/blob/master/images/promo.png?raw=true" />

### Почему это случилось

Хабр делает очень узкий круг людей. Их мало, а людей с их проблемами очень много. Если назвать себя "новым хабром" (что довольно некрасиво 😣), люди просто уйдут с платформы, что повлечёт за собой убытки. Мой проект задумывался как небольшой протест, стимул и пример для разработчиков основной площадки (а вообще мне важно было для себя хорошую читалку сделать). Делая смену названия, я отдаляюсь от основной площадки и перестаю нарушать авторские права, как бы это смешно не звучало. К тому же, в скором времени нас будет поддерживать RUVDS, а у них есть риск быть "выкинутым" с Хабра, за то, что они спонсорят такой проект.

### Какие будут последствия

Никаких, кроме того, что нужно будет переустановить PWA приложение (чтобы поменялось название) и поменять адрес в закладках 😇

Перед переустановкой можно экспортировать все свои настройки (Настройки -> Импорт настроек, для появления пункта в настройках возможно потребуется обновление). На новом сайте зайдите в **Настройки -> Импорт настроек** и вставьте сгенерированную строку в поле.

Приложение по старому адресу (habra.vercel.app) будет доступно, но оно будет перекидывать пользователя на новый URL - [geekr.vercel.app](https://geekr.vercel.app)

Как переустановить приложение: удалите его из системы и в браузере снова добавьте его под новым именем.

### А я хочу обратно, "хабра" звучит лучше

Без проблем - форкай проект и поменяй в `src/blocks/AppBar.tsx` и `src/blocks/SideNavigationDrawer.tsx` **geekr.** на **habra.**

---------

Спасибо за понимание, ваш @jarvis394.

**Comment by jarvis394** [comment]
Фига я умный: чтобы скопировать настройки, надо зайти на habra.vercel.app, а оно перекидывает на geekr.vercel.app.

Самого себя запутал 😶

**Comment by jakerdy** [comment]
Привет. Большое спасибо тебе за чудесный проект. 
Если не секрет, много ли людей ходят на хабру через geekr? Хотя бы порядок. Интерфейс реально супер удобный получился, и вот интересно сколько нас таких "адептов"..

**Reply by jarvis394** [reply]
<p align="center"><img src="https://user-images.githubusercontent.com/37776763/135479463-388f3d8a-5580-44ef-b5dc-9d913972a3d9.png"  width="300px" />
<img src="https://user-images.githubusercontent.com/37776763/135479598-172c0b1d-2767-47ba-b855-702068f3818c.png" width="200px" />
</p>

Около 400-700 сессий в день. Вполне вполне, я думаю 😊
